models/contacts.js

```diff
@@ -1,14 +1,76 @@
-// const fs = require('fs/promises')
+const { v4: uuidv4 } = require("uuid");
+const fs = require("fs").promises;
+const path = require("path");
+const contactsPath = path.join(__dirname, "contacts.json");
 
-const listContacts = async () => {}
+const listContacts = async () => {
+  try {
+    const contactsData = await fs.readFile(contactsPath);
+    const contacts = JSON.parse(contactsData);
+    return contacts;
+  } catch (err) {
+    console.error('Error reading contacts in listContacts:', err);
+    throw err;
+  };
+};
 
-const getContactById = async (contactId) => {}
+const getContactById = async (contactId) => {
+  try{
+    const contacts = await listContacts();
+    const contact = contacts.find(contact => contact.id === contactId);
+    return contact || null;
```

Comment:
jesli contact nie zostanie znaleziony to bedzie undefined. I generalnie undefined i null to jest to samo w tym wypadku - brak kontaktu :)